<!DOCTYPE html><!--  This site was created in Webflow. https://www.webflow.com  -->
<!--  Last Published: Tue Oct 11 2022 21:53:36 GMT+0000 (Coordinated Universal Time)  -->
<html data-wf-page="633efba2f46b6187bc8807a5" data-wf-site="633efba2f46b613c608807a0">
<head>
  <meta charset="utf-8">
  <title>Agency Management</title>
  <meta content="Agency Management" property="og:title">
  <meta content="Agency Management" property="twitter:title">
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="css/normalize.css" rel="stylesheet" type="text/css">
  <link href="css/webflow.css" rel="stylesheet" type="text/css">
  <link href="css/dbtl-2.webflow.css" rel="stylesheet" type="text/css">
  <!-- [if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js" type="text/javascript"></script><![endif] -->
  <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
  <link href="images/favicon.ico" rel="shortcut icon" type="image/x-icon">
  <link href="images/webclip.png" rel="apple-touch-icon">
</head>
<body>
  <div class="infographic-wrapper">
    <div data-current="build_wet_1" data-easing="ease" data-duration-in="0" data-duration-out="0" class="cycle1_wet w-tabs">
      <div class="tabs-menu w-tab-menu">
        <a data-w-tab="build_wet_1" class="w-inline-block w-tab-link w--current">
          <div>Scope</div>
        </a>
        <a data-w-tab="test_wet_1" class="w-inline-block w-tab-link">
          <div>Review</div>
        </a>
        <a data-w-tab="learn_wet_1" class="w-inline-block w-tab-link">
          <div>Reco</div>
        </a>
        <a data-w-tab="design_wet_1" class="w-inline-block w-tab-link">
          <div>Agreement</div>
        </a>
      </div>
      <div class="tabs-content w-tab-content">
        <div data-w-tab="build_wet_1" class="infographic-tab wet-1 w-tab-pane w--tab-active">
          <div class="tab-flex"><img height="" src="images/build6.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium build">Build </h3>
              <div class="text-block-4">Given our intent to try out different linkers to understand whether they could increase chitinase activity, Sporadicate’s cloning strategy involved the amplification of the anchor protein CotG via a high fidelity PCR (Phusion). Notably, the linker sequence was to be introduced at the 3’ end of the CotG CDS via carefully designed primer overhangs.  We used benchling to build and test our primers in silico and they did not seem to cause any issues. Having ordered the in silico designs, we ran amplification reactions for CotG and CotG-Linker.</div>
            </div>
          </div>
        </div>
        <div data-w-tab="test_wet_1" class="infographic-tab wet1 w-tab-pane">
          <div class="tab-flex"><img src="images/test3.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium">Test</h3>
              <div class="text-block-4">The PCR products were ran on an agarose gel; while a band for CotG-Linker was present, the CotG linear fragment failed to be amplified.</div><img sizes="223.99307250976562px" width="224" srcset="images/5-p-500.png 500w, images/5-p-800.png 800w, images/5.png 858w" src="images/5.png" loading="lazy" alt="" class="image">
            </div>
          </div>
        </div>
        <div data-w-tab="learn_wet_1" class="infographic-tab wet1 w-tab-pane">
          <div class="tab-flex"><img src="images/learn4.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium learn">Learn</h3>
              <div class="text-block-4">The results from this experiment led us to suppose that the primer pair employed for the amplification of CotG had a melting temperature which differed from what obtained on benchling. Before proceeding further, we tried a touchdown PCR to take annealing temperature differences into account. The results, again, were negative. We noticed that our CotG primer had a strong tendency to form dimers. It was time to try something else…</div>
            </div>
          </div>
        </div>
        <div data-w-tab="design_wet_1" class="infographic-tab wet1 w-tab-pane">
          <div class="tab-flex"><img src="images/design2.svg" width="1000" height="700" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium design">Design</h3>
              <div class="text-block-4">Once the sequences and display proteins were selected, two cloning strategies entailing the coding sequences of both a direct CotG-ChiS fusion and a CotG-Linker-ChiS design were considered for gene synthesis via IDT. However, this initial approach proved to be unviable due to the size and manufacturing complexity of the desired fusion proteins, which would have eaten up a significant proportion of our free IDT bases. Moreover, it lacked modularity, one characteristic we were seeking to allow for characterisation of ChiS activity with different anchor proteins or linkers. Hence, we settled on ordering ChiS as two linear fragments with custom overhangs harnessing type IIs restriction sites for an orderly assembly. For what concerns the anchors, it was decided to amplify them from genome via PCR.</div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="infographic-buttons">
      <a href="#" class="button-scope tab-button build-wet1 w-button">Scope</a>
      <a href="#" class="button-review tab-button w-button">Review</a>
      <a href="#" class="button-reco tab-button learn-wet1 w-button">Reco</a>
      <a href="#" class="button-agreement tab-button design-wet-1 w-button">Agreement</a>
    </div>
  </div>
  <div class="infographic-wrapper info-2">
    <div data-current="build_2_wet" data-easing="ease" data-duration-in="0" data-duration-out="0" class="cycle2_wet w-tabs">
      <div class="tabs-menu w-tab-menu">
        <a data-w-tab="build_2_wet" class="w-inline-block w-tab-link w--current">
          <div>Scope</div>
        </a>
        <a data-w-tab="test_2_wet" class="w-inline-block w-tab-link">
          <div>Review</div>
        </a>
        <a data-w-tab="learn_2_wet" class="w-inline-block w-tab-link">
          <div>Reco</div>
        </a>
        <a data-w-tab="design_2_wet" class="w-inline-block w-tab-link">
          <div>Agreement</div>
        </a>
      </div>
      <div class="tabs-content w-tab-content">
        <div data-w-tab="build_2_wet" class="infographic-tab w-tab-pane w--tab-active">
          <div class="tab-flex"><img height="" src="images/build6.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium build">Build </h3>
              <div class="text-block-4">We ran a gel with CotG-Linker as a control. As you can see in image x, CotG was successfully amplified. We gel purified the correct bands and decided to try out a level zero assembly with the Chitinase fragments.</div><img sizes="(max-width: 479px) 100vw, 500.0000305175781px" srcset="images/32-p-500.png 500w, images/32-p-800.png 800w, images/32.png 1000w" loading="lazy" src="images/32.png" alt="" class="image-2">
            </div>
          </div>
        </div>
        <div data-w-tab="test_2_wet" class="infographic-tab test w-tab-pane">
          <div class="tab-flex"><img src="images/test3.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium">Test</h3>
              <div class="text-block-4">We trasformed the golden gate reaction into E. coli TOP10 cells, and plated onto LB Cm. In order to assess the outcome of the assembly, colonies were screened with colony PCR, with successful clones being sent for Sanger sequencing.</div><img sizes="(max-width: 479px) 100vw, 500.0000305175781px" srcset="images/51-p-500.png 500w, images/51-p-800.png 800w, images/51-p-1080.png 1080w, images/51.png 1160w" loading="lazy" src="images/51.png" alt="" class="image-3">
            </div>
          </div>
        </div>
        <div data-w-tab="learn_2_wet" class="infographic-tab learn w-tab-pane">
          <div class="tab-flex"><img src="images/learn4.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium learn">Learn</h3>
              <div class="text-block-4">The trasformation reaction resulted in poor colony number and viability. Out of the limited number of positive clones, none passed our sequencing screening and were hence discarded in further assemblies. Interestingly, most of the mutations we identified arose in the ChiS coding region. After a couple other attempts with poor trasformation efficacy, we hypothesized that the Chitinase linear fragments were the likely cause of the assembly issues, as the CotG sequences appearead unmutated in sequencing rounds.</div>
            </div>
          </div>
        </div>
        <div data-w-tab="design_2_wet" class="infographic-tab design w-tab-pane">
          <div class="tab-flex"><img src="images/design2.svg" width="1000" height="700" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium design">Design</h3>
              <div class="text-block-4">Our primer design relied on a common Forward oligo for both CotG and CotG-Linker, and reverse sequences which targeted the 3’ end of the CotG anchor and only differed in the presence of a terminal overhang. In order to mitigate for the tendency of our CotG primer pair to form dimers, we tested five different concentrations of oligonucleotides (2µM /4 µM /6 µM /8µM /10 µM). Furthermore, we decided to take into account melting temperature differences, and used the autodelta function of our machine, which decreased Tm by one degree after each extension cycle.</div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="infographic-buttons">
      <a href="#" class="button-scope tab-button tab-button-active w-button">Scope</a>
      <a href="#" class="button-review tab-button w-button">Review</a>
      <a href="#" class="button-reco tab-button w-button">Reco</a>
      <a href="#" class="button-agreement tab-button w-button">Agreement</a>
    </div>
  </div>
  <div class="infographic-wrapper">
    <div data-current="Agreement" data-easing="ease" data-duration-in="0" data-duration-out="0" class="cycle3_wet w-tabs">
      <div class="tabs-menu w-tab-menu">
        <a data-w-tab="Scope" class="w-inline-block w-tab-link">
          <div>Scope</div>
        </a>
        <a data-w-tab="Review" class="w-inline-block w-tab-link">
          <div>Review</div>
        </a>
        <a data-w-tab="Reco" class="w-inline-block w-tab-link">
          <div>Reco</div>
        </a>
        <a data-w-tab="Agreement" class="w-inline-block w-tab-link w--current">
          <div>Agreement</div>
        </a>
      </div>
      <div class="tabs-content w-tab-content">
        <div data-w-tab="Scope" class="infographic-tab w-tab-pane">
          <div class="tab-flex"><img height="" src="images/build6.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium build">Build </h3>
              <div class="text-block-4">We performed another high fidelity PCR to amplify the Chitinase part1 and part 2 fragments. Both samples showed intense DNA bands. We extracted the samples from the gel and set up two assemblies into YTK in what effectively was a sub-zero cloning of each chitinase fragment. We trasformed the constructs into E. coli. (37 image)</div><img src="images/37.png" loading="lazy" srcset="images/37-p-500.png 500w, images/37-p-800.png 800w, images/37-p-1080.png 1080w, images/37.png 1244w" sizes="(max-width: 479px) 100vw, 500.0000305175781px" alt="" class="image-4">
            </div>
          </div>
        </div>
        <div data-w-tab="Review" class="infographic-tab test w-tab-pane">
          <div class="tab-flex"><img src="images/test3.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium">Test</h3>
              <div class="text-block-4">The transformation plates showed numerous colonies. We performed a verification PCR and sequenced the positive clones.</div><img src="images/40.png" loading="lazy" alt="" class="image-5">
            </div>
          </div>
        </div>
        <div data-w-tab="Reco" class="infographic-tab learn w-tab-pane">
          <div class="tab-flex"><img src="images/learn4.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium learn">Learn</h3>
              <div class="text-block-4">The sequencing results showed successful storage of each part into their respective YTK vector. This led us to understand that the STK toolkit was not tailered for the simultaneous assembly of many linear fragments in a one pot reaction, and that our initial design was not optimised for a swift assembly.Indeed, in Cycle 2 we documented the presence of multiple mutations in the Chitinase region of Level 0 STK clones. Nevertheless, in Cycle 3 we successfully stored each Chitinase fragment into a destination plasmid without reporting any mutations. Hence, we hypothesize that these changes arose at the moment of assembly.</div>
            </div>
          </div>
        </div>
        <div data-w-tab="Agreement" class="infographic-tab design w-tab-pane w--tab-active">
          <div class="tab-flex"><img src="images/design2.svg" width="1000" height="700" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium design">Design</h3>
              <div class="text-block-4">Given the negative results, it was time to rethink the design and assembly strategy. After chatting with our instructor Mo, he suggested to try and store the chitinase linear fragments (ChiS_part1 and ChiS_part2) into a “storage” vector. This would serve two purposes: first, it would allow us to cheaply source DNA in the lab via cultivation of a bacterial overnight culture trasformed with the storage plasmids. Secondly, we thought it could solve some of the issues associate with linear fragments, as circular DNA is more stable than linear DNA.<br><br>Given that our host lab works with Yarrowia lipolytica, we decided to select YTK001, an entry level vector into the YTK Golden Gate toolkit, as our storage system. To that purpose, we designed new primer pairs for each Chitinase part, with the addition of overhangs for BsmBI to facilitate assembly into YTK001. Notably, every new design was first created and tested in silico (Benchling) before proceeding to laboratory work.</div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="infographic-buttons">
      <a href="#" class="button-scope tab-button tab-button-active w-button">Scope</a>
      <a href="#" class="button-review tab-button w-button">Review</a>
      <a href="#" class="button-reco tab-button w-button">Reco</a>
      <a href="#" class="button-agreement tab-button w-button">Agreement</a>
    </div>
  </div>
  <div class="infographic-wrapper">
    <div data-current="Review" data-easing="ease" data-duration-in="0" data-duration-out="0" class="cycle4_wet w-tabs">
      <div class="tabs-menu w-tab-menu">
        <a data-w-tab="Scope" class="w-inline-block w-tab-link">
          <div>Scope</div>
        </a>
        <a data-w-tab="Review" class="w-inline-block w-tab-link w--current">
          <div>Review</div>
        </a>
        <a data-w-tab="Reco" class="w-inline-block w-tab-link">
          <div>Reco</div>
        </a>
        <a data-w-tab="Agreement" class="w-inline-block w-tab-link">
          <div>Agreement</div>
        </a>
      </div>
      <div class="tabs-content w-tab-content">
        <div data-w-tab="Scope" class="infographic-tab w-tab-pane">
          <div class="tab-flex"><img height="" src="images/build6.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium build">Build </h3>
              <div class="text-block-4">A Golden Gate reaction was performed overnight with both CotG and CotG linker anchor sequences. The assembled products were again trasformed into E. coli cells and selection occured on LB Cm.</div>
            </div>
          </div>
        </div>
        <div data-w-tab="Review" class="infographic-tab test w-tab-pane w--tab-active">
          <div class="tab-flex"><img src="images/test3.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium">Test</h3>
              <div class="text-block-4">When compared to our initial attempt, the trasformation plates displayed numerous colonies with few fluorescening. Colony PCR and restriction digestion with BsaI. We obtained successful clones and sent them for sequencing.</div><img src="images/48.png" loading="lazy" srcset="images/48-p-500.png 500w, images/48.png 688w" sizes="(max-width: 479px) 100vw, 500.0000305175781px" alt="" class="image-6">
            </div>
          </div>
        </div>
        <div data-w-tab="Reco" class="infographic-tab learn w-tab-pane">
          <div class="tab-flex"><img src="images/learn4.svg" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium learn">Learn</h3>
              <div class="text-block-4">This engineering cycle allowed us to look back at our initial design choices and understand their limitations; in particular, we will avoid linear fragments in STK Golden Gate reactions, as they are prone to instability.  The engineering principles we adopted across this learning process allowed us to pinpoint and understand the limitations of the toolkit we are working with, generating useful feedback for its implementation and use from future igem teams and researchers.</div>
            </div>
          </div>
        </div>
        <div data-w-tab="Agreement" class="infographic-tab design w-tab-pane">
          <div class="tab-flex"><img src="images/design2.svg" width="1000" height="700" alt="" class="infographic-image">
            <div class="infographic-text">
              <h3 class="heading-medium design">Design</h3>
              <div class="text-block-4">After having successfully secured ChiS part 1 and ChiS part 2 into the storage YTK plasmid, it was now time to proceed to the Level 0 assembly of our protein fusions.  At this stage, we planned on testing again the STK toolkit capabilities in a one pot reaction, hoping to see improvements due to the novel chitinase storage system. Importantly, every assembly had been performed in silico during the design stage of cycle 3 to assess the efficacy of our storage system.</div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="infographic-buttons">
      <a href="#" class="button-scope tab-button tab-button-active w-button">Scope</a>
      <a href="#" class="button-review tab-button w-button">Review</a>
      <a href="#" class="button-reco tab-button w-button">Reco</a>
      <a href="#" class="button-agreement tab-button w-button">Agreement</a>
    </div>
  </div>
  <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.5.1.min.dc5e7f18c8.js?site=633efba2f46b613c608807a0" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
  <script src="js/webflow.js" type="text/javascript"></script>
  <!-- [if lte IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/placeholders/3.0.2/placeholders.min.js"></script><![endif] -->
  <script type="text/javascript">
$(".tab-button").click(function(e) { 
  e.preventDefault();
  $(".tab-button").removeClass("tab-button-active");
  $(".w-tab-link:contains(" + e.target.innerText + ")").click();
  $(e.target).addClass("tab-button-active");
})
</script>
</body>
</html>